### ISSUE TITLE
Sideways/slanted text highlighting

### ISSUE THREAD

**Issue by emaadparacha** [Azure-Samples/azure-search-power-skills#29]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> N/A

### Any log messages given by the failure
> N/A

### Expected/desired behavior
> When I search for a specific text, usually the results come with that text highlighted on the specific document. I understand this is achieved with hOCR, but it only highlights text that is horizontal. How can I have highlight enabled (or is it possible) if there is slanted text or sideways text? I can still search for that text, and it shows in the transcript, but there is no highlight on the document itself. Is that possible?

### OS and Version?
> any

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by bleroy** [comment]
I'm not sure this is something we can do anything about in this repository. Can you give some additional information?

**Comment by emaadparacha** [comment]
@bleroy The question was more so related to hOCR, if the bounding boxes can be made around rotated text? Like if there's text rotated 90 degrees, can bounding boxes be made around them? Right now its only possible if the text runs left to right and is upright. If that makes sense?

**Comment by bleroy** [comment]
I don't think so: the skill is just translating information from one description schema to another, but it's not transforming it or doing any actual work. Unless there's information that exists in the source and that we're losing in translation?

**Comment by emaadparacha** [comment]
Oh no I don't mean can it tilt text, I mean if there is tilted text, can it create a bounding box around that? Like if the text were to process this image:

<img width="57" alt="Screen Shot 2020-11-05 at 21 36 49" src="https://user-images.githubusercontent.com/38843305/98319597-195a0580-1faf-11eb-9b3c-45a61b365f29.png">

Can it detect the text and create bounding boxes around the detected words?

I hope that make sense

**Comment by gmndrg** [comment]
Closing old issue that was either addressed at the time or native skills support the capability vs. a power skill.
